### DISCUSSION TITLE
Requesting clarification regarding Fluentd's support Policy

### DISCUSSION THREAD

**Discussion by mooneeb** [fluent/fluentd#4297] [Q&A] [answered]
Hello! 👋

According to the fluentd daemonset for k8s documentation here: https://github.com/fluent/fluentd-kubernetes-daemonset#image-versions, it's mentioned that fluentd `v1.x` is considered stable? I wanted to ask for clarity as to what does `v1.x` mean here. Does it mean that all minor versions are considered supported (1.1, 1.2, 1.3, ..., 1.16) and fixes will be backported? Or does it mean that only specific minor versions are supported (latest, n, n-1, n-2 etc)? AKA Which versions are supported at a given time? 

Your help would be very much appreciated 😃

**Comment by shazib-summar** [comment, marked as answer]
The supported versions are at this link: https://github.com/fluent/fluentd/blob/master/SECURITY.md

Related discussion: https://github.com/fluent/fluentd/discussions/4549
